.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+enableTelemetry: false
```

Comment:
Going along with the gitignore comment above, `.yarn/releases` should be committed so that developers don't have to re-download yarn. In that I think you also want at least `yarnPath` here as well, which refers to the downloaded version of Yarn in `.yarn/releases. You might also want `logFilters` as well to cut down on some of the output. See: https://github.com/MetaMask/metamask-module-template/blob/62bb1756ef6458580450a656dd7498c1610e8bcb/.yarnrc.yml